chromium_src/ui/base/accelerators/accelerator.cc

```diff
@@ -5,11 +5,57 @@
 
 #include "ui/base/accelerators/accelerator.h"
 
+#include <string>
+
+#include "components/grit/brave_components_strings.h"
```

Comment:
Do we need to add a dependency for this to ui/base/BUILD.gn?

Reply:
> Okay, this took me quite a while to work out - gn_check was passing, which was a good sign:

I think I missed this. AFAIK, `gn_check` doesn't check deps for files under`chromium_src`.
So, we have to be careful whenever including new headers in `chroium_src` cc @goodov 
Our rule is adding all deps in BUILD.gn directly. so I think maybe we should have patching in `ui/base/BUILD.gn` something like this?
```
deps += brave_chromium_src_ui_base_deps
```